docs/src/threadpool.rst

```diff
@@ -12,7 +12,11 @@ Its default size is 4, but it can be changed at startup time by setting the
 ``UV_THREADPOOL_SIZE`` environment variable to any value (the absolute maximum
 is 128).
 
-The threadpool is global and shared across all event loops.
+The threadpool is global and shared across all event loops. When a particular
+function makes use of the threadpool (i.e. when using :c:func:uv_queue_work)
```

Comment:
The correct syntax is:

~~~~
:c:func:`uv_queue_work`
~~~~

Note the backticks.